src/REST/Section/Project.php

```diff
@@ -31,6 +31,21 @@ public function get($project, array $expand = [])
         return $this->Jira->get("project/{$project}", $parameters);
     }
 
+    /**
+     * Returns all projects which are visible for the currently logged in user.
+     * If no user is logged in, it returns the list of projects that are visible when using anonymous access.
+     *
+     * @see https://docs.atlassian.com/software/jira/docs/api/REST/7.6.1/#api/2/project
+     *
+     * @return array
```

Comment:
almost missed this: you probably return \stdClass[] here, right?

Reply:
You right=)